pillar/params.sls

```diff
@@ -1,13 +1,23 @@
 # the CIDR for cluster IPs (internal IPs for Pods)
 cluster_cidr:     '172.16.0.0/13'
 
-# The min and max subnet, and length for node allocations.
-# The defaults allow for 510 pod IPs per host (/23) allowing
-# for a maximum of 1024 hosts.
-cluster_cidr_min: '172.16.0.0'
-cluster_cidr_max: '172.23.255.255'
+# The size of the subnet allocated to each host.
+# Defaults to 24 (i.e. /24) unless Network was configured to be smaller than a /24 in which case it is one less than the network.
```

Comment:
Why it defaults to 24? `cluster_cidr_len` defaults to /23, no?

Reply:
Typo